### DISCUSSION TITLE
Deploy online

### DISCUSSION THREAD

**Discussion by beralc** [3Alan/DocsMind#9] [Q&A]
Hi,

I've managed to install this locally, but was wondering if it's possible to put this on a web server. If so, what are the differences when installing it?

**Comment by 3Alan** [comment]
You need to deploy frontend and backend applications separately based on the documentation of Vite and Flask. Additionally, Dockerization is WIP。

**Comment by beralc** [comment]
Thanks. I'm pretty new with this. Is there a way to get more detailed instructions?

**Comment by mnallamalli97** [comment]
Hey @3Alan , I agree with beralc. 
The instructions for putting this on the internet is quite confusing. Can you update the readme for detailed instructions for hosting this app online?
